bigframes/core/__init__.py

```diff
@@ -354,9 +354,6 @@ def unpivot(
         *,
         passthrough_columns: typing.Sequence[str] = (),
         index_col_ids: typing.Sequence[str] = ["index"],
-        dtype: typing.Union[
```

Comment:
This was added so we could mix compatible numerics together in a single `df.sum()` or similar aggregation. You would manually pass the float dtype here and it would cast everything to that type. However, this isn't necessary as we can find the common supertype of the inputs and coerce to that automatically.